packages/manager/apps/catalog-revamp/src/pages/index.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { useQuery, UseQueryResult } from '@tanstack/react-query';
+import { OsdsDivider } from '@ovhcloud/ods-stencil/components/react/';
 import {
   getManagerHubCatalogList,
   getManagerHubCatalogListQueryKey,
 } from '@/api';
+import Loading from '@/components/Loading/Loading';
 import Error from '@/components/Error/Error';
+import SearchBar from '@/components/SearchBar/SearchBar';
 
 interface ServiceData {
   status: number;
   data: any;
 }
 
-export default function CatalogRevamp() {
+export default function CatalogReact() {
   const { t } = useTranslation('catalog-revamp');
 
   const service: UseQueryResult<ServiceData> = useQuery({
```

Comment:
we can move this query inside the useCatalog

Reply:
Done